### PR TITLE
eclipse/xtext-xtend#655 Fix exception.

### PR DESCRIPTION
**Pull request by ArneDeutsch**
Fix exception in case a file has already organized imports.

Signed-off-by: Arne Deutsch <Arne.Deutsch@itemis.de>

### CODE REVIEW

**Comment by ArneDeutsch** [comment]
This is only a fix as small part of the ticket. We should merge it right away, even if we do not implement the full feature rigt away.

File: org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java [hunk 1]
```diff
@@ -75,7 +75,7 @@
 					}
 					TextEdit textEdit = replaceConverter.convertToTextEdit(replace);
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java]: Why does it produce a null value?

Comment (tivervac) [reply org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java]: If replace.isEmpty then textEdit is null

Comment (tivervac) [reply org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java]: Don't think it matters a lot

Comment (ArneDeutsch) [reply org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java]: Adapted as suggested. Makes sense.

**Pull request merged**